system-test/read-rows.js

```diff
@@ -131,20 +130,23 @@ describe('Bigtable/Table', () => {
           .on('end', () => (endCalled = true))
           .on('error', err => (error = err));
         clock.runAll();
-        if (test.error) {
-          assert(!endCalled, `.on('end') should not have been invoked`);
-          assert.strictEqual(error.code, test.error);
-        } else {
-          assert(endCalled, `.on('end') shoud have been invoked`);
-          assert.ifError(error);
-        }
-        assert.deepStrictEqual(rowKeysRead, test.row_keys_read);
-        assert.strictEqual(
-          responses.length,
-          0,
-          'not all the responses were used'
-        );
-        assert.deepStrictEqual(requestedOptions, test.request_options);
+
+        setImmediate(function() {
```

Comment:
I'm missed this in the initial review, but putting this in a setImmediate after the call to `clock.runAll()` doesn't actually run this block of code in the test. It looks like read row retries are not working as expected since we're now getting grpc error codes and not http error code (since we're not wrapped by a GRPC service layer). I'll enable this block of code and send a proper PR to fix reties soon